code/modules/mob/living/carbon/xenomorph/embryo.dm

```diff
@@ -12,6 +12,7 @@
 	var/counter = 0
 	///How long before the larva is kicked out, * SSobj wait
 	var/larva_autoburst_countdown = 20
+	var/boost_timer = 0
```

Comment:
autodoc

Reply:
This autodoc is fine (I would generally avoid mentioning a specific number though, as that could change in the future and then people forget to update autodoc).